spec/features/addresses_controller_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper'
+
+describe AddressesController, js: true do
+  it 'finds address, fills form fields and finds number' do
```

Comment:
Wahrscheinlich könnten man noch gemeinsamen Setup Code in einem `before` block zusammenfassen